Filter/DateFilter.php

```diff
@@ -11,19 +11,15 @@
 
 namespace Sonata\AdminSearchBundle\Filter;
 
+use Sonata\AdminBundle\Form\Type\Filter\DateType;
+
 class DateFilter extends AbstractDateFilter
 {
     /**
-     * This filter has no range.
-     *
-     * @var bool
-     */
-    protected $range = false;
-
-    /**
-     * This filter does not allow filtering by time.
-     *
-     * @var bool
+     * {@inheritdoc}
      */
-    protected $time = false;
```

Comment:
This is a BC-break, please restore the properties, and see if they are changed in the constructor, and throw if they are.